identity/src/main/java/com/stripe/android/identity/ui/ErrorScreen.kt

```diff
@@ -0,0 +1,133 @@
+package com.stripe.android.identity.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.google.android.material.composethemeadapter.MdcTheme
+import com.stripe.android.identity.R
+
+@Composable
+internal fun ErrorScreen(
+    title: String,
+    message1: String? = null,
+    message2: String? = null,
+    topButton: Pair<String, () -> Unit>? = null,
+    bottomButton: Pair<String, () -> Unit>? = null,
+) {
+    MdcTheme {
+        Column(
+            modifier = Modifier
```

Comment:
Can you add the Modifier as a parameter with a default value of `Modifier`, and then use it at the top-most level?